web/src/engine/dom-utils/src/stylesheets.ts

```diff
@@ -44,19 +39,24 @@ export class StylesheetManager {
       return;
     }
 
-    if(typeof(fd['files']) == 'undefined') {
-      fd['files']=fd['source'];
-    }
-    if(typeof(fd['files']) == 'undefined') {
+    // if(typeof(fd.files) == 'undefined') {
+    //   fd.files=fd['source'];
+    // }
+    if(typeof(fd.files == 'undefined')) {
```

Comment:
This looks like a change in behaviour; also commented code here should be removed.

Reply:
The `KeyboardProperties` type converts the other two formats to the one that specifies `.files` in advance.

https://github.com/keymanapp/keyman/blob/1081fb605294987b9ce3b621c4cc95f9f0e1fda3/common/web/keyboard-processor/src/keyboards/keyboardProperties.ts#L102-L109

https://github.com/keymanapp/keyman/blob/1081fb605294987b9ce3b621c4cc95f9f0e1fda3/common/web/keyboard-processor/src/keyboards/keyboardProperties.ts#L3-L9

https://github.com/keymanapp/keyman/blob/1081fb605294987b9ce3b621c4cc95f9f0e1fda3/web/src/engine/dom-utils/src/stylesheets.ts#L2

So... the behavior still exists; it was just centralized elsewhere.